items/de.json

```
@@ -0,0 +1,202 @@
+{
+  "42": "42",
+  "air": "Luft",
+  "alchemistry": "Alchemie",
+  "algae": "Algen",
+  "alien": "Alien",
+  "animal": "Tier",
+  "aquarium": "Aquarium",
+  "archipelago": "Archipel",
+  "ash": "Asche",
+  "astronaut": "Astronaut",
+  "atoll": "Atoll",
+  "axe": "Axt",
+  "baby": "Baby",
+  "beach": "Strand",
+  "beach_castle": "Sandburg",
+  "beverage": "Trinken",
+  "binary_system": "Binäres System",
+  "bird": "Vogel",
+  "black_hole": "Schwarzes Loch",
+  "blood": "Blut",
+  "boat": "Boot",
+  "bone": "Knochen",
+  "book": "Buch",
+  "box": "Kiste",
+  "bread": "Brot",
+  "breaded_fish": "Fischbrötchen",
+  "brick": "Ziegel",
+  "building": "Gebäude",
+  "cable": "Kabel",
+  "cake": "Kuchen",
+  "camera": "Kamera",
+  "campfire": "Lagerfeuer",
+  "carbon_dioxide": "Kohlendioxid",
+  "card": "Spielkarte",
+  "cardboard": "Pappe",
+  "carrier": "Flugzeugträger",
+  "castle": "Burg",
+  "cat": "Katze",
+  "chainsaw": "Kettensäge",
+  "chalet": "Chalet",
+  "charcoal": "Holzohle",
+  "cheese": "Käse",
+  "chicken": "Huhn",
+  "child": "Kind",
+  "christmas_sleigh": "Weihnachtlicher Schlitten",
+  "city": "Stadt",
+  "clay": "Lehm",
+  "clock": "Uhr",
+  "cloud": "Wolke",
+  "coal": "Kohle",
+  "continent": "Kontinent",
+  "coral": "Koralle",
+  "cow": "Kuh",
+  "death": "Tod",
+  "desert": "Wüste",
+  "diamond": "Diamant",
+  "dinosaur": "Dinosaurier",
+  "dirt": "Dreck",
+  "dog": "Hund",
+  "doghouse": "Hundehütte",
+  "dragon": "Drache",
+  "drone": "Drone",
+  "dyson_sphere": "Dyson-Sphäre",
+  "earth": "Die Erde",
+  "egg": "Ei",
+  "electricity": "Elektrizität",
+  "elsa": "Elsa",
+  "engine": "Motor",
+  "eye": "Auge",
+  "factory": "Fabrik",
+  "fairy_tail": "Märchen",
+  "farmer": "Bauer",
+  "feather": "Feder",
+  "fir": "Tanne",
+  "fire": "Feuer",
+  "fish": "Fisch",
+  "fish_and_chips": "Fisch und Pommes",
+  "flour": "Mehl",
+  "flower": "Blume",
+  "food": "Essen",
+  "forest": "Wald",
+  "french_fries": "Pommes",
+  "fried_egg": "Gebratenes Ei",
+  "fruit": "Frucht",
+  "furnace": "Ofen",
+  "galaxy": "Galaxie",
+  "garden": "Garten",
+  "generator": "Generator",
+  "geyser": "Geysir",
+  "glass": "Glas",
+  "glasses": "Brille",
+  "gold": "Gold",
+  "grass": "Grass",
+  "greenhouse": "Glashaus",
+  "halo": "Heiligenschein",
```

```suggestion
  "halo": "Halo",
```
Here's the Halo from the game